index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title></title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <section id="main-content">
+      <h1>
+        <p>Welcome to my Cookie Stand!</p>
+      </h1>
+    </section>
+    <script src="app.js" type="text/javascript"></script>
```

Comment:
This file doesn't have anything with the id 'cookie-list', so your code breaks if you link to it in this file.